src/Entity/Diff/PropertyDiffer.php

```diff
@@ -79,9 +79,9 @@ private function diffPropertyArrays( array $from, array $to ) {
 	private function toDiffArray( Property $property ) {
```

Comment:
I concur a new class is not warranted at this point. And I agree a Fingerprint could be passed to this method, though find this an odd observation given the -2.